src/Helper/Helper.php

```diff
@@ -304,6 +305,17 @@ public function removeRelations(array $entries): void {
     }
   }
 
+  /**
+   * Cleanup relations.
+   *
+   * @param \Drupal\webform\WebformSubmissionInterface|null $submission
+   *   A webform submission.
+   */
+  public function cleanUp(WebformSubmissionInterface $submission = NULL): void {
+    $relations = $this->getDetachedQueueSubmissionRelations($submission->id());
```

Comment:
This will fail is `$submission` is `null`:

```suggestion
    $relations = $this->getDetachedQueueSubmissionRelations($submission?->id());
```